src/store/actions/Room.ts

```diff
@@ -1,20 +1,18 @@
-import { ToggleButtonGroup } from "react-bootstrap";
-
 export const PREVIEW_ROOM = "PREVIEW_ROOM";
 export const EXIT_PREVIEW_ROOM = "EXIT_PREVIEW_ROOM";
 export const TOGGLE_MUTE_REACTIONS = "TOGGLE_MUTE_REACTIONS";
 
-interface ExitRoomAction {
+export interface ExitRoomAction {
   type: typeof EXIT_PREVIEW_ROOM;
 }
 
-interface PreviewRoomAction {
+export interface PreviewRoomAction {
   type: typeof PREVIEW_ROOM;
   room: string;
 }
 
-interface ToggleMuteAction {
-  type: typeof ToggleButtonGroup;
+export interface ToggleMuteAction {
+  type: typeof TOGGLE_MUTE_REACTIONS;
 }
```

Comment:
[note] I'm not sure this was ever a valid type for this to have. In any case, this part of the code doesn't even seem to be used at the moment, so making this change shouldn't affect anything. I didn't delete it outright because moving to more of a Redux-based pattern for muting is potentially an ideal direction to go in.